extensions/amp-consent/0.1/amp-consent.js

```diff
@@ -434,6 +422,34 @@ export class AmpConsent extends AMP.BaseElement {
           }
         });
   }
+
+  /**
+   * Handles the display of postPromptUI
+   */
+  handlePostPromptUI_() {
+    this.notificationUiManager_.onQueueEmpty(() => {
+      if (!this.postPromptUI_) {
+        return;
+      }
+      if (!this.uiInit_) {
+        this.uiInit_ = true;
+        toggle(this.element, true);
```

Comment:
These functions that update the elements' CSS should be wrapped in a vsync mutate.

Reply:
Question: This is a position fixed element? Do we still need to wrap that?

Reply:
Hmm I think so. /cc @aghassemi ?

Reply:
Got it. I think it's always best to wrap css change in vsync mutate. However we might have failed to do that with many components 😢

Reply:
yes, mutating a position fixed element will cause a smaller relayout (container within just the position fixed layer) but it is a relayout anyway and better to be in sync with others ones.

Reply:
fixed.